### PR TITLE
Organize build args for readability/mergeability

### PR DESCRIPTION
**Pull request by dagood**
I like splitting args across lines because:

* Diffs (especially 3-way for merging) are hard to read with long lines.
* Automatic diffing can be smarter when the parts are grouped like this.
* Actual conflicts are more likely to be resolvable using basic choose A/B/C.

In a few cases I was able to share some parameters that were duplicated before.

### CODE REVIEW

File: repos/nuget-client.proj [hunk 1]
```diff
@@ -26,16 +26,32 @@
...
 
-    <Exec Command="$(DotnetToolCommand) msbuild $(ProjectDirectory)/build/build.proj /t:BuildXPLAT /p:VisualStudioVerion=15.0 /p:Configuration=$(Configuration) /p:BuildRTM=false /p:BuildNumber=$(NuGetClientBuildNumber) /bl:build.binlog $(RedirectRepoOutputToLog)"
+    <Exec Command="$(BuildCommandBase) /t:BuildXPLAT /bl:build.binlog $(RedirectRepoOutputToLog)"
           EnvironmentVariables="@(EnvironmentVariables)"
           WorkingDirectory="$(ProjectDirectory)" />
 
-    <Exec Command="$(DotnetToolCommand) msbuild $(ProjectDirectory)/build/build.proj /t:PackXPLAT /p:VisualStudioVerion=15.0 /p:Configuration=$(Configuration) /p:BuildRTM=false /p:BuildNumber=$(NuGetClientBuildNumber) /p:PackageOutputPath=$(PackagesOutput) /p:NoPackageAnalysis=true /flp:v=detailed /bl:pack.binlog $(RedirectRepoOutputToLog)"
+    <PropertyGroup>
+      <PackCommand>$(BuildCommandBase) /t:PackXPlat</PackCommand>
+      <PackCommand>$(PackCommand) /p:PackagOutputPath=$(PackagesOutput)</PackCommand>
```
Comment (crummel) [inline repos/nuget-client.proj]: Typo: `PackagOutputPath` instead of `PackageOutputPath`.

Comment (dagood) [reply repos/nuget-client.proj]: Nice spot, wonder how I managed that.

This is a weird property though, come to think of it... telling NuGet to put packages in `$(ProjectDirectory)artifacts/nupkgs`, but that's where NuGet would put them anyway. (Which is why the prebuilt baselines didn't change.) I'll just fix it for now, anyway--maybe it lets us be more flexible between NuGet submodule commits or something.

**Comment by dagood** [comment]
@dotnet-bot test Ubuntu16.04 Release
`504 (Gateway Timeout)` accessing nuget.org... I filed https://github.com/dotnet/core-eng/issues/4214 for an earlier case, but it went away. Hopefully this will too.

```
  Failed to download package 'System.Reflection.TypeExtensions.4.1.0' from 'https://api.nuget.org/v3-flatcontainer/system.reflection.typeextensions/4.1.0/system.reflection.typeextensions.4.1.0.nupkg'.
  Response status code does not indicate success: 504 (Gateway Timeout).
```

@dotnet-bot test OSX10.12 Release
`java.nio.channels.ClosedChannelException`

**Comment by dagood** [comment]
@dotnet-bot test OSX10.12 Release
`java.nio.channels.ClosedChannelException`

**Pull request merged**